### PR TITLE
feat: add styles for modal and form user-menu

### PR DESCRIPTION
**Pull request by GusBaamonde**
Standard modal sizes creation and form for user-menu

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -3,21 +3,36 @@ $("body").on({
...
+$("body").on('click', '#close-menu', function() {
   $('.header-main').removeClass('open');
-});
\ No newline at end of file
+});
+
+// modals
+
+$("#btn-modal-large").click(function() {
+  $("#modal-large").fadeIn(500);
```
Comment (cbernat) [inline assets/js/main.js]: Maybe add this default delay into a var `500` What do you think?

Comment (GusBaamonde) [reply assets/js/main.js]: great idea!

**Comment by GusBaamonde** [comment]
> Good work Gus



> Good work Gus

Thanks Nacho!

**Pull request merged**